test/test_twodnavierstokes.jl

```diff
@@ -109,28 +118,36 @@ function test_twodnavierstokes_deterministicforcingbudgets(dev::Device=CPU(); n=
   TwoDNavierStokes.set_zeta!(prob, 0*x)
 
   E = Diagnostic(TwoDNavierStokes.energy,      prob, nsteps=nt)
-  D = Diagnostic(TwoDNavierStokes.dissipation, prob, nsteps=nt)
-  R = Diagnostic(TwoDNavierStokes.drag,        prob, nsteps=nt)
-  W = Diagnostic(TwoDNavierStokes.work,        prob, nsteps=nt)
-  diags = [E, D, W, R]
+  D = Diagnostic(TwoDNavierStokes.energy_dissipation, prob, nsteps=nt)
+  R = Diagnostic(TwoDNavierStokes.energy_drag,        prob, nsteps=nt)
+  W = Diagnostic(TwoDNavierStokes.energy_work,        prob, nsteps=nt)
+  Z = Diagnostic(TwoDNavierStokes.enstrophy,      prob, nsteps=nt)
+  DZ = Diagnostic(TwoDNavierStokes.enstrophy_dissipation, prob, nsteps=nt)
+  RZ = Diagnostic(TwoDNavierStokes.enstrophy_drag,        prob, nsteps=nt)
+  WZ = Diagnostic(TwoDNavierStokes.enstrophy_work,        prob, nsteps=nt)
+  diags = [E, D, W, R, Z, DZ, WZ, RZ]
 
   # Step forward
   stepforward!(prob, diags, nt)
   TwoDNavierStokes.updatevars!(prob)
 
-  E, D, W, R = diags
+  E, D, W, R, Z, DZ, WZ, RZ = diags
   t = round(μ*cl.t, digits=2)
 
   i₀ = 1
   dEdt = (E[(i₀+1):E.i] - E[i₀:E.i-1])/cl.dt
-  ii  = (i₀):E.i-1
+  dZdt = (Z[(i₀+1):Z.i] - Z[i₀:Z.i-1])/cl.dt
+  ii = (i₀):E.i-1
   ii2 = (i₀+1):E.i
 
   # dEdt = W - D - R?
   total = W[ii2] - D[ii] - R[ii]
+  totalZ = WZ[ii2] - DZ[ii] - RZ[ii]
   residual = dEdt - total
+  residualZ = dZdt - totalZ
 
   isapprox(mean(abs.(residual)), 0, atol=1e-8)
+  isapprox(mean(abs.(residualZ)), 0, atol=1e-8)
```

Comment:
The way the test is written now will only return `true` or `false` for the enstrophy budget.
Last two lines of the `tests should be changed to 

```julia
return isapprox(mean(abs.(residual)), 0, atol=1e-8) & isapprox(mean(abs.(residualZ)), 0, atol=1e-8)
```